.github/pull_request_template.md

```diff
@@ -4,6 +4,6 @@
 
 ## Checklist before requesting a review
 - [ ] I have performed a self-review of my code
-- [ ] I have checked that [readthedocs](https://readthedocs.org/projects/credoai-lens/) API still is constructed correctly
+- [ ] I have checked that [readthedocs](https://readthedocs.org/projects/credoai-lens/) API documentation is constructed correctly
```

Comment:
This is sorta complicated since it will require activating the present branch on RTD and compiling. Will RTD auto-delete docs for deleted branches after they're merged?

If that's the process we want to follow, we should be explicit.